### ISSUE TITLE
Issues Connecting (Unexpected Failures)

### ISSUE THREAD

**Issue by Seek0999** [lavalink-devs/Lavalink#652]
## Description
I've tried using Java Version 18, 17 and now 13.

For some reason every time i connect to the Lavalink client, it always throws the same errors. Not sure what I'm doing wrong but it keeps throwing Unexpected failure erorr.

Range of errors I'm encountering: 

```
java.io.IOError: java.io.FileNotFoundException: Invalid file path

ERROR 10728 --- [  XNIO-1 task-1] io.undertow.request.io : UT005090: Unexpected failure

ERROR 10728 --- [  XNIO-1 task-2] io.undertow.request : UT005071: Undertow request failed HttpServerExchange{ GET /}
```

## Version info

**Client used:**

[Vulkava](https://www.npmjs.com/package/vulkava) - Version (Vulkava@1.2.4) | Latest at the time of issue 

Output of `java -version`:
```
openjdk version "13.0.11" 2022-04-19
OpenJDK Runtime Environment Zulu13.48+19-CA (build 13.0.11+4-MTS)
OpenJDK 64-Bit Server VM Zulu13.48+19-CA (build 13.0.11+4-MTS, mixed mode, sharing)
```

Lavalink Version:
```
Version:        3.4
Build:          1257
Build time:     26.10.2021 14:18:37 UTC
Branch          master
Commit:         a364833
Commit time:    26.10.2021 14:15:20 UTC
JVM:            13.0.11
Lavaplayer      1.3.96-original
```

**Comment by topi314** [comment]
Have you checked if it's a problem with vulkava?

**Comment by Seek0999** [comment]
> Have you checked if it's a problem with vulkava?

I've contacted the developer of the library and he said it isnt an issue with Vulkava, so im not sure...

**Comment by davidffa** [comment]
It seems to be a issue with windows or some windows java distributions.
Its not a problem with the lavalink client (vulkava), as I tested it by making a direct `GET /` request from the browser and happens the same problem.
I've tested on a **windows** machine with java 11.0.15 (zulu), java 13.0.11 (zulu), java 11.0.14 (oracle) and java 17.0.2 (oracle), but the error only happens using the java 11 and 13 **from zulu**.
Here is the full stacktrace:
```
2022-04-29 20:14:01.410 ERROR 32312 --- [  XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /[192.168.1.81:61748](http://192.168.1.81:61748/)
2022-04-29 20:14:01.431 ERROR 32312 --- [  XNIO-1 task-1] io.undertow.request                      : UT005071: Undertow request failed HttpServerExchange{ GET /}

java.io.IOError: java.io.FileNotFoundException: Invalid file path
        at org.xnio.channels.Channels$4.run(Channels.java:998) ~[xnio-api-3.3.8.Final.jar!/:3.3.8.Final]
        at org.xnio.channels.Channels$4.run(Channels.java:988) ~[xnio-api-3.3.8.Final.jar!/:3.3.8.Final]
        at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]
        at org.xnio.channels.Channels.<clinit>(Channels.java:988) ~[xnio-api-3.3.8.Final.jar!/:3.3.8.Final]
        at io.undertow.servlet.spec.ServletOutputStreamImpl.close(ServletOutputStreamImpl.java:618) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.spec.HttpServletResponseImpl.closeStreamAndWriter(HttpServletResponseImpl.java:528) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.spec.HttpServletResponseImpl.responseDone(HttpServletResponseImpl.java:617) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:328) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:78) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:133) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:130) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:249) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:78) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:99) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.Connectors.executeRootHandler(Connectors.java:376) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:830) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
        at java.base/java.lang.Thread.run(Thread.java:829) ~[na:na]
Caused by: java.io.FileNotFoundException: Invalid file path
        at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:231) ~[na:na]
        at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:126) ~[na:na]
        at org.xnio.channels.Channels$4.run(Channels.java:993) ~[xnio-api-3.3.8.Final.jar!/:3.3.8.Final]
        ... 20 common frames omitted

2022-04-29 20:14:01.445 ERROR 32312 --- [  XNIO-1 task-2] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /[192.168.1.81:61749](http://192.168.1.81:61749/)
2022-04-29 20:14:01.447 ERROR 32312 --- [  XNIO-1 task-2] io.undertow.request                      : UT005071: Undertow request failed HttpServerExchange{ GET /}

java.lang.NoClassDefFoundError: Could not initialize class org.xnio.channels.Channels
        at io.undertow.servlet.spec.ServletOutputStreamImpl.close(ServletOutputStreamImpl.java:618) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.spec.HttpServletResponseImpl.closeStreamAndWriter(HttpServletResponseImpl.java:528) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.spec.HttpServletResponseImpl.responseDone(HttpServletResponseImpl.java:617) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:328) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:78) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:133) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:130) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:249) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:78) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:99) ~[undertow-servlet-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.Connectors.executeRootHandler(Connectors.java:376) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:830) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
        at java.base/java.lang.Thread.run(Thread.java:829) ~[na:na]
```

**Comment by Seek0999** [comment]
Resolved the issue,

Was due to 99% of the Java versions i installed happened to not be compatible.

**Comment by aikaterna** [comment]
> It seems to be a issue with windows or some windows java distributions. Its not a problem with the lavalink client (vulkava), as I tested it by making a direct `GET /` request from the browser and happens the same problem. I've tested on a **windows** machine with java 11.0.15 (zulu), java 13.0.11 (zulu), java 11.0.14 (oracle) and java 17.0.2 (oracle), but the error only happens using the java 11 and 13 **from zulu**. 

I have seen this from a user of mine this morning on a Windows 10 device running AdoptOpenJDK/Adoptium 11.0.15 as well.

Lavalink logs: https://gist.github.com/Cog-CreatorsBot/fbedc27d243684dc6743c66d2ec05d1e
Java info:
![image](https://user-images.githubusercontent.com/20862007/166266742-0d6bdcd7-5035-4899-a2fd-dd538945e69d.png)

**Comment by davidffa** [comment]
It also happens on java 17.0.3 from oracle, but the bug was fixed in versions 17.0.3.1 and 18.0.1.1

See:
https://www.oracle.com/java/technologies/javase/17-0-3-1-relnotes.html#JDK-8285660
https://bugs.openjdk.java.net/browse/JDK-8285445
